library/portable-simd/crates/core_simd/src/vector/ptr.rs

```diff
@@ -21,12 +24,16 @@ where
     #[inline]
     #[must_use]
     pub fn wrapping_add(self, addend: Simd<usize, LANES>) -> Self {
+        #[cfg(bootstrap)]
         // Safety: converting pointers to usize and vice-versa is safe
         // (even if using that pointer is not)
         unsafe {
             let x: Simd<usize, LANES> = mem::transmute_copy(&self);
             mem::transmute_copy(&{ x + (addend * Simd::splat(mem::size_of::<T>())) })
         }
+        #[cfg(not(bootstrap))]
+        // Safety: this intrinsic doesn't have a precondition
+        unsafe { intrinsics::simd_arith_offset(self, addend) }
```

Comment:
@workingjubilee I assume it's not a big deal porting this commit back to the portable SIMD repo?

Reply:
Nope, cherry-picking this should be easy, though if it's in a separate commit that would remove any possible awkwardness.

Reply:
I thought `got subtree` does this automatically?

But yeah I can split this in 2 commits easily.

Reply:
It gets Weirder in a repo of this size. :grimacing: